inc/form.class.php

```diff
@@ -1082,21 +1085,24 @@ protected function showMyLastForms() : void {
          echo '<div class="card-body text-center text-muted" >'.__('No form waiting for validation', 'formcreator').'</div>';
       } else {
          echo '<div class="card-body">';
-         echo '<ul>';
+         echo '<ul class="list-group">';
```

Comment:
class was missing; see the 1st card in showMyLastForms() (a few dozens line above)